code/obj/item/organs/tail.dm

```diff
@@ -20,6 +20,7 @@
 	var/failure_ability = "clumsy"	// The organ failure ability associated with this organ.
 	var/human_getting_monkeytail = 0	// If a human's getting a monkey tail
 	var/monkey_getting_humantail = 0	// If a monkey's getting a human tail
+	var/clothing_image_icon = null // if the tail has clothing sprites, set this to the appropriate icon
```

Comment:
change this to codedoc and if you want to change the rest

Reply:
Added in [0ca96b9](https://github.com/goonstation/goonstation/pull/9773/commits/0ca96b9e2da60a73de054e02e5e442837ff98a2e)